src/lib/accessibility/Behaviors/Popup/popupFocusTrapBehavior.ts

```diff
@@ -0,0 +1,17 @@
+import { Accessibility } from '../../types'
+import popupBehavior from './popupBehavior'
+import * as _ from 'lodash'
+
+/**
+ * @description
+ * Adds role='button' to 'trigger' component's part, if it is not focusable element and no role attribute provided.
+ * Adds tabIndex='0' to 'trigger' component's part, if it is not tabbable element and no tabIndex attribute provided.
+ * Adds attribute 'aria-disabled=true' to 'trigger' component's part based on the property 'disabled'.
+ * Traps focus inside component and by default allowing click outside this component.
```

Comment:
seems that `allowing click outside this component` is not checked by tests. maybe we should omit it for now - and leave only the statement that is verified by tests?

Reply:
Yes, it is not tested, and as we discussed it's done not only for tests. This is also what actually the users will read. So just a bit extended description, specifying the behavior. 
But ok, will remove it for now, as we don't have any places using different configurations with focus trap.